### PR TITLE
Vrinda 1.0/vrinda service connect1

### PR DESCRIPTION
**Pull request by vrindavnair**
created a signin page and otp page

### CODE REVIEW

File: src/SignInOTPComponents/Pages/ServiceProviderSignIn.jsx [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+    {
+      name: 'password',
+      type: 'password',
+      placeholder: 'Password',
+      label: 'Password',
+      required: true,
+    },
+  ];
+
+  const inputConfig={};
```
Comment (JaseelaAlimahamed) [inline src/SignInOTPComponents/Pages/ServiceProviderSignIn.jsx]: give input config

File: src/components/reUsableComponents/FormComponent.jsx [hunk 2]
```diff
@@ -53,12 +53,12 @@ const FormComponent = ({ fieldConfigs, buttonConfig, inputConfig, apiEndpoint, h
       return schema;
     }, {})
   );
-  
+
   return (
     <div className="bg-light-gray p-6 max-w-xl w-full md:max-w-lg lg:max-w-lg lg:p-4 xl:max-w-md xl:p-4">
       {heading && (
-        <div className="bg-dark-gray p-4 rounded-t-lg w-full mx-auto">
-          <h2 className="text-white font-heading text-center text-lg font-bold">{heading}</h2>
```
Comment (JaseelaAlimahamed) [inline src/components/reUsableComponents/FormComponent.jsx:61]: you are not giving heading then why you change heading design, make it old one

File: src/routes/ProviderRoutes.jsx [hunk 3]
```diff
@@ -1,14 +1,19 @@
...
 
 
 
 function ProviderRoutes() {
   return (
     <div>
       <Routes>
-        
-        <Route path="/" element={<Navigate to="/signin" />} />
+           <Route path="/" element={<ServiceProviderSignIn />} />
```
Comment (JaseelaAlimahamed) [inline src/routes/ProviderRoutes.jsx]: for sign in give path/signin

File: src/components/reUsableComponents/FormComponent.jsx (hunk 2 above)
Comment (vrindavnair) [reply src/components/reUsableComponents/FormComponent.jsx:61]: Implemented as requested.

File: src/SignInOTPComponents/Pages/ServiceProviderSignIn.jsx (hunk 1 above)
Comment (vrindavnair) [reply src/SignInOTPComponents/Pages/ServiceProviderSignIn.jsx]: Implemented as requested.

File: src/components/reUsableComponents/FormComponent.jsx (hunk 2 above)
Comment (vrindavnair) [reply src/components/reUsableComponents/FormComponent.jsx:61]: Implemented as requested.

File: src/routes/ProviderRoutes.jsx (hunk 3 above)
Comment (vrindavnair) [reply src/routes/ProviderRoutes.jsx]: Implemented as requested.

**Review by JaseelaAlimahamed — APPROVED** [review]

**Pull request merged**